server/Grid.js

```diff
@@ -1,12 +1,17 @@
-import { CellTypes } from "./TetrisConsts.js";
+import { CellType } from "./TetrisConsts.js";
 
 export default class Grid {
+  /**
+   * @param {import("./TetrisConsts.js").GridArray} array
```

Comment:
I'm not saying you should change it, but more so you know, you can define a type in a comment, and then reference it later. eg:

```js
/** @typedef {import("./TetrisConsts.js").GridArray} GridArray */

export default class Grid {
  /**
   * @param {GridArray} array
   ...
   ```